Runtime/Behaviours/CinemachineBlendListCamera.cs

```diff
@@ -278,12 +278,14 @@ private void OnGuiHandler()
                 CinemachineDebug.ReleaseScreenPos(this);
             else
             {
+#if CINEMACHINE_UNITY_IMGUI
```

Comment:
This entire method should be removed if !CINEMACHINE_UNITY_IMGUI.  What's left is bizarre.